CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 ## Solidus 3.2.0.alpha (master, unreleased)
 
+- Promotions with a `match_policy` of `any` are deprecated.
```

Comment:
Maybe explain the usage of the new setting here?

Reply:
Done!